docs/arch.plantuml

```diff
@@ -0,0 +1,38 @@
+@startuml
+center header <b>Ceres Rollup Service Architecture
+queue "unified.metrics.json topic" as input
+card Metrics
+note right: com.rackspacecloud.metrics.rollup.domain.Metric
+node KTable [
+<b> KTable
+==========
+5m Tumbling Window
+==========
+20m Tumbling Window
+==========
+60m Tumbling Window
+==========
+240m Tumbling Window
+==========
+1440m Tumbling Window
+] 
+note right: com.rackspacecloud.metrics.rollup.config.TopologyConfiguration
+card Rollups
+note right: com.rackspacecloud.metrics.rollup.domain.RolledUp
+note bottom: rollups contain: min, max, mean, sum, count for the corresponding window
+queue 5m
+note left: these are all kafka topics
+queue 20m
+queue 60m
+queue 240m
+queue 1440m
+
+input --> Metrics
+Metrics --> KTable : grouped by key:  accountType.account.monitoringSystem.collectionName.device.deviceLabel.collectionLabel
```

Comment:
I can't remember what each of these values would be but `deviceLabel` sounds worrying.

Reply:
do we need to create a jira ticket to fix that?

Reply:
That seems like a good idea

Reply:
I added this ticket for it:
https://jira.rax.io/browse/CERES-317